sql/updates/world/3.3.5/2020_08_xx_xx_world.sql

```diff
@@ -0,0 +1,10 @@
+UPDATE `command` SET `help`='Syntax: .debug transport [1/0]
```

Comment:
I don't like `[1/0]` here. Use `start/stop`.

Reply:
Also, _what_ transport? current transport?

Reply:
> Also, _what_ transport? current transport?

Yes, the one you're on. I'll clarify that bit.